modules/sdk-coin-avaxp/src/avaxp.ts

```diff
@@ -90,6 +92,32 @@ export class AvaxP extends BaseCoin {
           }
         }
         break;
+      case TransactionType.Import:
+        if (explainedTx.outputs.length !== 1) {
+          throw new Error('Expected 1 C-chain address');
+        }
+
+        if (params.txParams?.recipients) {
+          if (
+            explainedTx.outputs[0].address !== params.txParams.recipients[0].address ||
+            !isValidAddress(explainedTx.outputs[0].address)
+          ) {
+            throw new Error('C-Chain address does not match with expected txParams');
+          }
+
+          const fee = new BigNumber(explainedTx.fee.fee);
+          const outputAmount = new BigNumber(explainedTx.outputs[0].amount);
+          const totalOutputAmount = new BigNumber(explainedTx.outputAmount);
+
+          if (!outputAmount.plus(fee).isEqualTo(params.txParams.recipients[0].amount)) {
+            throw new Error('C-Chain amount does not match with expected txParams');
+          }
+
+          if (!totalOutputAmount.plus(fee).isEqualTo(params.txParams.recipients[0].amount)) {
+            throw new Error('C-Chain output amount does not match with expected txParams');
```

Comment:
let's update the throw message as i mentioned in https://github.com/BitGo/bitgo-microservices/pull/25063 ty